Pipfile

```diff
@@ -15,6 +15,8 @@ python-dotenv = "*"
 flask-talisman = "*"
 Flask-Babel = "*"
 Jinja2 = "*"
+requests = "*"
+xmltodict = "*"
```

Comment:
There is no need to add these as they are part of the Python standard library. Try it by typing `python 3` then `import requests`...no error message!